src/Shared/CommunicationsUtilities.cs

```diff
@@ -154,55 +154,6 @@ private static int FileVersionHash
         [DllImport("kernel32.dll", EntryPoint = "RtlMoveMemory")]
         internal static unsafe extern void CopyMemory(char* destination, char* source, uint length);
```

Comment:
Do you call this anymore?

Reply:
Indeed I don't. And that means I can remove the whole #define. That's fulfilling, right before going on leave!